test/test_ops.py

```diff
@@ -388,6 +466,101 @@ def test_vmapvjp(self, device, dtype, op):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(fn, args, {}):
                 self.assertEqual(loop_out, batched_out, atol=1e-4, rtol=1e-4)
 
+    # There are several variations we care about
+    # 1) primal batched (TODO)
+    # 2) tangent batched (batched grads) <--
+    # 3) both batched (TODO)
+    # The below tests (2) only.
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_vmapjvp', {
+        # These should to be fixed:
+
+        # Random failures: Copied from above
+        xfail('nn.functional.dropout'),  # randomness
+        xfail('double'),
+        xfail('linalg.householder_product'),
+
+        # Batching rule not implemented for aten::_local_scalar_dense
+        xfail('linalg.eigh'),
+        xfail('linalg.tensorinv'),
+        xfail('linalg.inv'),
+        xfail('linalg.matrix_power'),
+        xfail('linalg.cholesky'),
+
+        # Batching rule not implemented for aten::logical_{and,or}.out
+        xfail('cumprod'),
+        xfail('__rpow__'),
+        xfail('clamp', 'scalar'),
+        xfail('logit'),
+        xfail('fmin'),
+        xfail('fmax'),
+        xfail('float_power'),
+        xfail('pow'),
```

Comment:
To summarize, I think the things that need looking into from the functorch side are the following categories:
```
# Random failures: Copied from above
...

# Expected the output of forward differentiable view operations
# to have the tangent have the same layout as primal
...

# RuntimeError: Cannot access data pointer of Tensor that doesn't have storage
...
```

Reply:
Yes that would make sense. I've updated some faulty testing code as well as this list https://github.com/pytorch/pytorch/issues/66357 to reduced the discrepancy between the two. The remaining ones are:
- `Batching rule not implemented for aten::logical_{and,or}.out`
- `Batching rule not implemented for aten::_local_scalar_dense`
- `block_diag` and `diag_embed`

Reply:
Update: `block_diag` should be the last one. The others have been resolved!